packages/website/tsconfig.json

```diff
@@ -9,6 +9,11 @@
     "lib": ["DOM"],
     "noEmit": true,
     "noImplicitAny": false,
+    "resolveJsonModule": true,
+    "baseUrl": ".",
+    "paths": {
+      "@site/*": ["./*"]
```

Comment:
Oh I like this quite a lot... though `@` is (theoretically, at least) a valid npm org name and I've generally preferred `~/` as a prefix over `@` to avoid the ambiguity. Would be great if Docusaurus could support that...

Side tangent 1: I remember seeing a proposal for a lint rule somewhere around enforcing using aliased root-level imports like this one. If that works well I'd love to have us use it.

Side tangent 2: also https://github.com/lydell/eslint-plugin-simple-import-sort 👌 

(neither tangent is necessary at all for this PR, just spitballing!)

Reply:
we should use 
```
"extends": "@tsconfig/docusaurus/tsconfig.json"
```

https://www.npmjs.com/package/@tsconfig/docusaurus

Reply:
I actually don't particularly like extending that config. It's quite wonky in my editor and often can't be resolved... We don't have a whole lot of options anyways